apps/codeforafrica/src/payload/collections/Media.js

```diff
@@ -5,6 +5,7 @@ const Media = {
   },
   access: {
     read: () => true, // Everyone can read Media
+    create: () => true,
```

Comment:
? Why do we need this? We only want admins to be able to create and **not** _everyone_.

Reply:
@kilemensi We could not upload any media unless we added this.

Reply:
Is it there in charter.africa? Why can we upload there @kelvinkipruto? Again, I'm not against solutions but we need to know **_why_** of a think more than even the **_what_**.

Reply:
@kilemensi  I think it could be an issue with the database as it works well locally and only happens with the previews